File: Data Structures & Algorithms/two-integer-sum/submission-0.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hashmap = {}
        result = []
        for index in range(len(nums)):
            if target - nums[index] not in hashmap:
                hashmap[nums[index]] = index
            else:
                result.append(index)
                result.append(hashmap[target-nums[index]])
        result.sort()
        return result

File: Data Structures & Algorithms/find-minimum-in-rotated-sorted-array/submission-0.cpp
class Solution {
public:
    int findMin(vector<int> &nums) {
        int L = 0;
        int MIN = 0;
        int R = nums.size() - 1;
        int MID;
        while(L<R){
            if(nums[L] < nums[R]){
                return nums[L];
            }
            MID = (L + R)/2;
            if(nums[MID] > nums[R]){
                L = MID + 1;
            }
            else{
                R = MID;
            }
        }
        return nums[L];
        
    }
};
